### DISCUSSION TITLE
Search directories by name

### DISCUSSION THREAD

**Discussion by prov2433** [community/community#21894] [New to GitHub]
<p>I know I can search by filenames, using a query like:</p>
<p>org:holdgold12 filename:ice_cream</p>
<p>The results do not include directories named “ice_cream”.</p>
<p>How can I search for directories by name, or include directories in the above query?</p>

**Comment by nadiajoyce** [comment, marked as answer]
<p>Hi <a class="mention" href="/prov2433">@prov2433</a>,</p>
<p>Thanks for being part of the Community Forum! I’ll answer your question as best I can.</p>
<p>You can, indeed, search based on directory. Full instructions and details are <a href="https://help.github.com/articles/searching-code/#search-by-the-location-of-a-file-within-the-repository" rel="nofollow noopener">here</a>, but to do what you’re describing (return results with directories named “ice_cream”) you’d use:</p>
<p>org:holdgold12 path:ice_cream</p>
<p>There are several other options for further customizing your search, and I’d recommend reading the guide I linked to above to find out more.</p>
<p>Cheers!</p>

**Comment by jackmarki** [comment]
that's Informative and helpful
